graylog2-web-interface/src/views/logic/searchtypes/events/EventHandler.js

```diff
@@ -0,0 +1,105 @@
+// @flow strict
+import { groupBy } from 'lodash';
+import WidgetFormattingSettings from 'views/logic/aggregationbuilder/WidgetFormattingSettings';
+import type { ChartDefinition } from 'views/components/visualizations/ChartData';
+
+export type Event = {
+  id: string,
+  timestamp: string,
+  message: string,
+  alert: boolean,
+  streams: Array<string>,
+};
+
+export type Events = Array<Event>;
+
+type GroupedEvents = { [string]: Events };
+
+type Shape = {
+  type: 'line',
+  y0: number,
+  y1: number,
+  x0: string,
+  x1: string,
+  opacity: number,
+  line: {
+    color: string,
+  }
+};
+
+export type Shapes = Array<Shape>;
+const eventsDisplayName = 'Alerts';
+
+export default {
+  convert(events: Array<Event>) {
+    return events;
+  },
```

Comment:
Could you explain in which case we need the convert function?

Reply:
So this code, does not break: https://github.com/Graylog2/graylog2-server/blob/master/graylog2-web-interface/src/views/logic/QueryResult.js#L12